src/core/override-configuration.js

```diff
@@ -18,7 +18,25 @@ function overrideConfig(config) {
     .filter(([key, value]) => !!key && !!value)
     .map(([codedKey, codedValue]) => {
       const key = decodeURIComponent(codedKey);
-      const decodedValue = decodeURIComponent(codedValue.replace(/%3D/g, "="));
+      // The URL is not necessarily trusted, so we escape any suspicious tag-like
+      // content. While hyperHTML seems to block execution of inserted scripts (most
+      // likely because adding a script via innerHTML doesn't work), other harmful
+      // tags (think <style>) are enough to classify this as a minor XSS threat.
+      // Although harmful configuration would never occur to a user who clicks around,
+      // say, w3c.org, they might click a link to w3c.org from another website, see the
+      // trusted w3c domain in their address bar, but receive a misleading page because
+      // of the query parameters. Content on w3c.org should only be content that
+
+      // Even with this escaping, it is possible for an attacker to make a link that
+      // displays a page with misleading content. They might add themselves to the list
+      // of copyright holders, for example, a task which requires no HTML tags.
+
+      // Potential solutions include whitelisting certain options, blocking query
+      // string interpretation when `document.referrer` is present, or displaying
+      // a clear warning message to make it obvious that the page was modified.
+      const decodedValue = decodeURIComponent(codedValue.replace(/%3D/g, "="))
+        .replace(/>/g, "&gt;")
```

Comment:
After reading https://wonko.com/post/html-escaping I'm not so sure. If anything's going unescaped into an HTML attribute we would actually need to escape *more* characters. There's no reason ReSpec would be doing that, but I'll have to look around and make sure.

Reply:
Not sure I follow, can you give a concrete example of the attack?

Reply:
If the template is `<a href="${untrusted}">`, the attacker can provide `[" onmouseover="evil()]`.

Reply:
It seems like hyperHTML detects expansions in an attribute and unconditionally escapes it, so we only need to care about angled brackets.

Reply:
You are right. However, we then need to make sure that anything going through `conf` always goes through hyperHTML.

Reply:
(this is a separate/larger issue... but thought I'd mention it)

Reply:
We should also make a broader definition of the security model we are using. For example, if an attacker set wgURI they could link to a misleading site and it would appear as if that was the official site of the working group.

Reply:
yeah, good point... we should check that it's a w3c URL etc. would need to go through each configuration option and evaluate the risk of each one.